packages/alfa-angular/tsconfig.json

```diff
@@ -1,4 +1,7 @@
 {
+  "compilerOptions": {
+      "experimentalDecorators": true
+  },
```

Comment:
Is this change necessary? I don't see other updates to the `angular.spec.tsx`

Reply:
Indeed, not needed until we actually get to write a test that needs it.